charts/ecosystem/templates/couchdb-setup-job.yaml

```diff
@@ -7,7 +7,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: "{{ .Release.Name }}-couchdb-setup"
+  name: "{{ .Release.Name }}-couchdb-setup-{{ randAlphaNum 5 | lower }}"
```

Comment:
The couchdb setup job has a TTL of 2mins (see https://github.com/galasa-dev/helm/pull/98/files#diff-c8477b398f88bacfb42bea6a2230fb4f92bd43d8aad11e7478c15af4c391b716R12). However, if the `documentRevisionLimit` value is updated and a `helm upgrade` is issued before those 2mins are up, then the following error is thrown:

```
Error: UPGRADE FAILED: cannot patch "myservice-couchdb-setup" with kind Job: Job.batch "myservice-couchdb-setup" is invalid: spec.template: Invalid value: core.PodTemplateSpec{...}: field is immutable
```

Adding a random suffix alleviates this issue (of course, it doesn't completely eliminate the issue though).